### DISCUSSION TITLE
Forced Device reboot after Update run

### DISCUSSION THREAD

**Discussion by ORG-MBD** [Romanitho/Winget-AutoUpdate#637] [Q&A]
Hello,

I've enabled AutoUpdate on User Logon for everyone, so WAU runs when somebody logs into the Device. Now I have the situation where some Devices are rebooting 15 Minutes after logon. Is there some way to suppress the Reboot ?

I've searched for some ADMX Policy since im Using the WAU Intune fork. Couldn't find anything.

Hope someone can help me :)

**Comment by Romanitho** [comment]
Hi,
If you're using a fork, open a case in there repo.

WAU is working on top of Winget. We don't manage how applications handle reboots. I suggest you to blacklist app that causing unexpected reboots or try to use mods.
